### PR TITLE
fix-forms-bugs

### LINKED ISSUE #189: Fix All Forms Bugs & High Fidelity Forms
- Fix all bugs on the [Find Your Shrek](https://docs.google.com/document/d/1f3dqxzRCiNGZ2yWU_FtyrNNjUOFZ8lxiEuXo958NVDM/edit?pli=1#heading=h.pylkt3kz9rjv) doc that are related to general and admin forms.
- Implement changes in [FYA Change Log](https://docs.google.com/document/d/19Hsa-zn2we4xdwYrZHAFy-t16yhSkQdcOmvtLIB3b7s/edit#)
- Apply high fidelity to forms

### PR DESCRIPTION
**Pull request by ctc-devops**
- Fix all bugs on the [Find Your Shrek](https://docs.google.com/document/d/1f3dqxzRCiNGZ2yWU_FtyrNNjUOFZ8lxiEuXo958NVDM/edit?pli=1#heading=h.pylkt3kz9rjv) doc that are related to general and admin forms.
- Implement changes in [FYA Change Log](https://docs.google.com/document/d/19Hsa-zn2we4xdwYrZHAFy-t16yhSkQdcOmvtLIB3b7s/edit#)
- Apply high fidelity to forms
closes #189

### CODE REVIEW

**Review by claudeisakeeb — DISMISSED** [review]

**Comment by meghakak** [comment]
Fixed bugs from NPO:
1. No spacing between image and submit button once image uploaded
2. Copy to include with the Box Number image section: "In order to confirm your box, please include an image of the box number in the upper left hand corner of the box."
3. For launch method, can the methods please be: Given to someone, Launched at a location

File: src/components/RelocateBoxForm/RelocateBoxForm.jsx [hunk 1]
```diff
@@ -12,14 +12,14 @@ import {
   Input,
```
Comment (jennymylee) [inline src/components/RelocateBoxForm/RelocateBoxForm.jsx:12]: "Support for jpeg, jpg, png" should be centered.

File: src/components/PickupBoxForm/PickupBoxForm.jsx [hunk 2]
```diff
@@ -11,6 +11,7 @@ import {
   uploadBoxPhoto,
```
Comment (jennymylee) [inline src/components/PickupBoxForm/PickupBoxForm.jsx:11]: "Support for jpeg, jpg, png" should be centered.

File: src/components/AddBoxForm/AddBoxForm.jsx [hunk 3]
```diff
@@ -24,6 +24,7 @@ import {
   validateZip,
   validateBoxNumber,
   uploadBoxPhoto,
+  validateDate,
```
Comment (jennymylee) [inline src/components/AddBoxForm/AddBoxForm.jsx:27]: remove colon after Message subheading

**Review by jennymylee — CHANGES_REQUESTED** [review]
Just a few things. Good work! :)

**Review by claudeisakeeb — CHANGES_REQUESTED** [review]
same comments as jennymylee

File: src/components/RelocateBoxForm/RelocateBoxForm.jsx (hunk 1 above)
Comment (meghakak) [reply src/components/RelocateBoxForm/RelocateBoxForm.jsx:12]: done

File: src/components/PickupBoxForm/PickupBoxForm.jsx (hunk 2 above)
Comment (meghakak) [reply src/components/PickupBoxForm/PickupBoxForm.jsx:11]: done

File: src/components/AddBoxForm/AddBoxForm.jsx (hunk 3 above)
Comment (meghakak) [reply src/components/AddBoxForm/AddBoxForm.jsx:27]: yes ma'am

**Review by claudeisakeeb — APPROVED** [review]
handled

**Review by jennymylee — APPROVED** [review]
LGTM <3

**Pull request merged**